draft-irtf-cfrg-frost.md

```diff
@@ -674,35 +656,66 @@ A FROST ciphersuite must specify the underlying prime-order group details
 and cryptographic hash function. Each ciphersuite is denoted as (Group, Hash),
 e.g., (ristretto255, SHA-512). This section contains some ciphersuites.
 The RECOMMENDED ciphersuite is (ristretto255, SHA-512) {{recommended-suite}}.
+The (edwards25519, SHA-512) ciphersuite is included for backwards compatibility
+with {{!RFC8032}}.
+
+## FROST(edwards25519, SHA-512)
+
+This ciphersuite uses edwards25519 for the Group and SHA-512 for the Hash function `H`
+meant to produce signatures indistinguishable from Ed25519 as specified in {{!RFC8032}}.
+The value of the contextString parameter is empty.
+
+- Group: ed25519 {{!RFC8032}}
+  - SerializeElement: Implemented as specified in {{!RFC8032, Section 5.1.2}}.
+  - DeserializeElement: Implemented as specified in {{!RFC8032, Section 5.1.3}}.
+- Hash (`H`): SHA-512, and Nh = 64.
+  - H1(m): Implemented by computing H("rho" || m), interpreting the lower
+    32 bytes as a little-endian integer, and reducing the resulting integer modulo
+    L = 2^252+27742317777372353535851937790883648493.
+  - H2(m): Implemented by computing H(m), interpreting the lower 32 bytes
```

Comment:
Shouldn't you specify the input for `H2` as you did on line 672 for `H1`?

Reply:
Also, I seem to recall that the public key is also hashed into H2. Double checking that is not the case here.

Reply:
No -- this is specifically done for compatibility with RFC8032.